app/scrappers/GenericScrapper.scala

```diff
@@ -27,12 +27,14 @@ import org.mongodb.scala.model.Filters._
  * You need to override some things and you get a functional scrapper
  * for most of the imageboards.
  */
-abstract class GenericScrapper
-  (
-    implicit ws: WSClient,
-    mongo: Mongo,
-    ec: ExecutionContext,
-  ) extends Actor {
+abstract class GenericScrapper(
+  name: String,
+  baseUrl: String,
+  favicon: String
+)(implicit ws: WSClient,
```

Comment:
Holy shit, is this really how they do in Scala?
I mean ')('

Reply:
Also shouldn't `implicit ...` start on a new line?

Reply:
` I mean ')('`
No idea, they style guide doesn't say anyting on that, i guess i can move it on separate lines.
`Also shouldn't implicit ... start on a new line?`
Again, no idea, i start it on the same line.